antora-ui-camel/src/css/blog.css

```diff
@@ -223,3 +223,13 @@ article.blog p {
 .blog .page-item.active a {
   color: var(--color-white);
 }
+
+@media screen and (max-width: 1023px) {
+  article.blog:first-child {
```

Comment:
This selector might be a bit problematic as it might target anything (including an element that's absolutely positioned). I'm fairly certain that to accommodate the second row on the top level navigation we need to move or add the padding/margin globaly. I think this change needs to go to `base.css` and cover all pages. We should also base this from the `--navbar-height` variable like we do for the `--body-top`. Remember these variables can change and then `5rem` is not accurate.